openetl_utils/connector_utils.py

```
@@ -5,8 +5,8 @@
 home = os.environ['OPENETL_HOME']
 sys.path.append(home)
 import json
-from utils.database_utils import DatabaseUtils
-from utils.enums import *
+from openetl_utils.database_utils import DatabaseUtils
+from openetl_utils.enums import *
```

Ruff (0.8.2)

`from openetl_utils.enums import *` used; unable to detect undefined names

(F403)